firebase-firestore/src/main/java/com/google/firebase/firestore/core/SyncEngine.java

```diff
@@ -589,4 +595,19 @@ private boolean errorIsInteresting(Status error) {
 
     return false;
   }
+
+  private boolean isRetryableError(Exception e) {
```

Comment:
Since this is specific to transactions, it should probably be named isRetryableTransactionError() or similar.